frontend/src/app/components/transaction/transaction.component.ts

```diff
@@ -358,7 +358,7 @@ export class TransactionComponent implements OnInit, AfterViewInit, OnDestroy {
       }),
     ).subscribe((accelerationHistory) => {
       for (const acceleration of accelerationHistory) {
-        if (acceleration.txid === this.txId && (acceleration.status === 'completed' || acceleration.status === 'completed_provisional')) {
```

Comment:
Haven't tested yet, but I think this might cause the out of band fee to still display the fee delta after the transaction gets mined by a non participating pool (because then, `accelerationInfo` is not defined upon confirmation)

Reply:
> might cause the out of band fee to still display the fee delta

display the fee delta where?

Reply:
ah, I see.

in that case, I'd rather fix the root cause (either clear the `tx.feeDelta` once we know a transaction was not accelerated by a participating pool, or tighten up the template logic)